tox.ini

```
@@ -97,3 +97,15 @@ deps =
     -r{toxinidir}/tests/integration/requirements.txt
 commands =
     pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
+
+[testenv:src-docs]
+allowlist_externals=sh
+setenv =
+    PYTHONPATH=src
```

```suggestion
    PYTHONPATH = {toxinidir}/lib:{[vars]src_path}
```
I think you need to include the libs in `PYTHONPATH`. See also [here](https://chat.canonical.com/canonical/pl/h6zp5g5sxtrttb8i99dezxzbbh).

Or don't set the PYTHONPATH and it should be passed by the `[testenv]` configuration above.